db/scripts/V5__add_image_snapshot.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE element_instance ADD image_snapshot	TEXT;
+ALTER TABLE iofog_change_tracking ADD `is_image_snapshot` BIGINT;
```

Comment:
better to use TINYINT(1)

Reply:
change `is_image_snapshot` to `image_snapshot`